<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADDStudentWindow</class>
 <widget class="QWidget" name="ADDStudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление студента</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/add-user</normaloff>:/add-user</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="FIO_textEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Оценки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_1">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_3">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_4">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_5">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_add">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_exit">
       <property name="text">
        <string>Выйти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
